### ISSUE TITLE
Font suggestion for edit form

### ISSUE THREAD

**Issue by gamesbook** [darklow/django-suit#600]
Django version: 1.10.7
Django Suit version: v2a1 
Python version: 3.5

Suggestion - get font sizes to match in the default interface.  The buttons on the right have extra large text compared to the actual form and menu (the colors also seem out of sync but this is less important).

![admin_edit](https://cloud.githubusercontent.com/assets/354027/24961677/ead60b8c-1f99-11e7-9f10-aa581341f5a3.png)

**Comment by darklow** [comment]
I like larger buttons, that was meant intentional, easier to click, harder to miss :)

**Comment by gamesbook** [comment]
I am OK with the buttons being large; its just the font that seems out-of-place.

**Comment by marc-gist** [comment]
I agree, buttons and text too large.  I would also like to see the white space between them reduced. or perhaps there is an easy way to customize these per site?

**Comment by darklow** [comment]
@marc-gist Extending `base_site.html` and adding custom `admin.css` for any extra CSS you need is the right way. 

Example in demo: https://github.com/darklow/django-suit/blob/v2/demo/demo/templates/admin/base_site.html#L8

**Comment by darklow** [comment]
From my user experience I suggest removing extra Save buttons unless they are really necessary, like adding 1000x items per day. I always leave just one for my projects. Otherwise every time they make me think too much what do I want to click right now.

**Comment by marc-gist** [comment]
@darklow base_suite.html is not used in the code base, but you have it modified in the demo?

**Comment by darklow** [comment]
@marc-gist `base_site.html` is official way of customising Django admin:
https://docs.djangoproject.com/en/1.11/intro/tutorial07/

**Comment by marc-gist** [comment]
why in your code do you do this: inside "admin/base_site.html"?

`{% extends 'admin/base.html' %}`
`{# Workaround to avoid circular template extends errors #}`

**Comment by darklow** [comment]
You should look at the demo app, not suit code. Once you use your own `base_site.html` you will override whatever suit has. You can read docs from v1 about extending `base_site.html`: http://django-suit.readthedocs.io/en/develop/configuration.html#templates

**Comment by marc-gist** [comment]
No offese, but I'm not interested, or payed for the demo app, I paid for the code, and do not want to override unless i have to.  I.e. it should just by installing the app into Django, which is does beautifully.  Now I want to override some things, which should be easy.  therefore, I goto the source, as I would for native django code, and override only what I absolutely have to.

**Comment by darklow** [comment]
Extending `base_site.html` is the official Django way of extending admin, not sure what more official way to extend you are looking for. Once you project evolves you will have to have custom CSS, custom JS and other templates files, so nothing wrong with that. How about footer content, are you sure you want to keep Django Suit copyright and links to Django suit instead of adding your own things? Can't think of simpler way rather than extending using one simple file which is documented in Django Suit and Django documentation.

**Comment by marc-gist** [comment]
@darklow thanks for the quick response. Clearly I need to read more :) and will experiment more.  great project!
